src/ray/raylet/worker_pool.h

```diff
@@ -701,7 +701,22 @@ class WorkerPool : public WorkerPoolInterface, public IOWorkerPoolInterface {
   /// - Add URI reference when a worker instance registered.
   /// - Remove URI reference when a worker instance disconnected.
   ///
-  /// An example:
+  /// A normal state change flow is:
+  ///   job level:
+  ///       HandleJobStarted(ref + 1 = 1) -> HandleJobFinshed(ref - 1 = 0)
+  ///   worker level:
+  ///       StartWorkerProcess(ref + 1 = 1)
+  ///       -> OnWorkerStarted X 3 (ref + 3 = 4)
```

Comment:
What does X3 mean here?

Reply:
three workers are started.

Reply:
So this assumes there are 3 workers per process? Can you mention it somewhere?

Reply:
Add a note here https://github.com/ray-project/ray/pull/20789/commits/884188c1b27d58e412b2f172fd5d7cb7c3e704eb#diff-a88735bc6aa09dd8248a3e73081916b5096a79c4bb7de91f9875d2c5a84ef075R726